.cirrus.yml

```diff
@@ -81,7 +81,7 @@ standalone_ubuntu_task:
     - rm spicy-dev.deb
 
   build_script:
-    - (mkdir build && cd build && cmake -DCMAKE_INSTALL_PREFIX=/opt/spicy-plugin -G Ninja .. && ninja -j4)
+    - (mkdir build && cd build && cmake -DCMAKE_INSTALL_PREFIX=/opt/spicy-plugin -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -G Ninja .. && ninja -j4)
```

Comment:
The commit says "if possible": does this mean that cmake will ignore the `CMAKE_CXX_COMPILER_LAUNCHER` setting if `ccache` isn't found? Or will it bail out? (Just wondering, either way is fine I think because we control the setting and know its installed)

Reply:
We currently do not set up ccache e.g., for installations via `zkg`. This should be also be possible (e.g., by replace the compiler invoked by e.g., `spicyc` with a symlink to `ccache` in the container image), but I didn't attempt it here.

Reply:
Ok, probably just misunderstood your comment in the commit message.